src/main/resources/db/changelog/20231003113535-entity_layering_schema.xml

```diff
@@ -373,8 +373,10 @@
         <validCheckSum>9:16c11c8edc80574ab030cd5fbcec5a07</validCheckSum>
         <!-- 4.4.10 -->
         <validCheckSum>9:56c16786668c16c345e8ce64ae890851</validCheckSum>
-        <!-- 4.4.11+ -->
+        <!-- 4.4.11 -->
         <validCheckSum>9:d11ca759108c7919ce364d946c825460</validCheckSum>
+        <!-- 4.5.2+ -->
```

Comment:
once we backport this to candlepin-4.4-HOTFIX, I assume this should say 4.4.21+ right?